tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -147,7 +147,8 @@ void InternalCommandsHandler::preExecute(IRequestsHandler::ExecutionRequest &req
                                          concordUtils::SpanWrapper &parent_span) {
   if (req.flags & bftEngine::DB_CHECKPOINT_FLAG) return;
 
-  if (req.outExecutionStatus != static_cast<uint32_t>(OperationResult::UNKNOWN)) {
+  if (req.outExecutionStatus != static_cast<uint32_t>(OperationResult::UNKNOWN) &&
+      (req.flags & HAS_PRE_PROCESSED_FLAG)) {
```

Comment:
I don't think this additional condition is correct. We never call this function twice, like it was before with execute() function.